build.gradle

```diff
@@ -5,10 +5,14 @@ buildscript {
     repositories {
         google()
         jcenter()
+        // For publishing files to Bintray
+        maven { url 'https://oss.jfrog.org/artifactory/oss-snapshot-local/' }
```

Comment:
This line is not necessary to publish to arttifactory. Could you remove it?

Reply:
Removed